tests/qiskit_convert_test.py

```diff
@@ -1091,3 +1094,24 @@ def test_RealAmplitudes_numeric_params() -> None:
     unitary2 = tkc2.get_unitary()
     assert compare_unitaries(qiskit_unitary, unitary1)
     assert compare_unitaries(unitary1, unitary2)
+
+
+# https://github.com/CQCL/pytket-qiskit/issues/256
+def test_symbolic_param_conv() -> None:
```

Comment:
I note that this test does not have an assert statement or anything of that kind.

Is the idea just to ensure that this code executes successfully and doesn't give the 
```
TypeError: EvalfMixin.evalf() got an unexpected keyword argument 'real'
```

As before?

Reply:
Yes that was my idea, I was thinking about checking stuff in the final circuit, but I would expect the way qiskit is setting up the circuit or the compilation works might change without generating any issues here, so I decided not to do any additional checks.